### PR TITLE
update action list to use one binder

### PR DESCRIPTION
**Pull request by abhishek1508**
### Description

Fixes https://github.com/mapbox/navigation-sdks/issues/1525

### CODE REVIEW

File: libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/recenter/RecenterButtonComponent.kt [hunk 1]
```diff
@@ -1,40 +1,50 @@
...
+                when (navigationState) {
+                    is NavigationState.RoutePreview -> {
+                        recenterButton.isVisible = false
+                    }
+                    else -> {
+                        recenterButton.isVisible = cameraState.cameraMode == TargetCameraMode.Idle
+                    }
+                }
+            }.collect()
+        }
```
Comment (kmadsen) [inline libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/recenter/RecenterButtonComponent.kt:47]:
```suggestion
        coroutineScope.launch {
            combine(
                cameraViewModel.state,
                navigationStateViewModel.state
            ) { cameraState, navigationState ->
                navigationState == NavigationState.RoutePreview || cameraState.cameraMode == TargetCameraMode.Idle
                }
            }.collect { isVisible ->
                recenterButton.isVisible = isVisible
            }
        }
```

Comment (kmadsen) [reply libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/recenter/RecenterButtonComponent.kt:47]: boolean logic on my refactor may be wrong. But the suggestion here is to make the "collect" block update the recenterButton visibility

**Review by kmadsen — APPROVED** [review]
Minor comment but looks good!

File: libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/recenter/RecenterButtonComponent.kt (hunk 1 above)
Comment (abhishek1508) [reply libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/recenter/RecenterButtonComponent.kt]: Fixed.

**Pull request merged**